dark/reads.py

```diff
@@ -190,13 +190,23 @@ class AARead(Read):
     """
     def properties(self):
         """
-        Translate an amino acid sequence to properties.
+        Translate an amino acid sequence to properties of the form:
+        'F': HYDROPHOBIC | AROMATIC.
 
         @return: A generator yielding properties for the residues in the
             current sequence.
         """
         return (PROPERTIES.get(aa, NONE) for aa in self.sequence)
 
+    def propertyDetails(self):
+        """
+        Translate an amino acid sequence to properties. Each property of the
+        amino acid gets a value scaled from -1 to 1.
+
+        @return: A list of property dictionaries.
+        """
+        return (PROPERTY_DETAILS.get(aa, NONE) for aa in self.sequence)
```

Comment:
You don't need the second argument to `get` here, as `None` is the default. I think I may have introduced that, in the `properties` method, so you could delete it there as well, too. Thanks :-)